packages/core/src/dialects/sqlite/query-interface.js

```diff
@@ -369,4 +353,23 @@ export class SqliteQueryInterface extends AbstractQueryInterface {
 
     return `${prefix}_${suffix}`;
   }
+
+  /**
+   * @override
+   */
+  async bulkDelete(table, where, options, model) {
```

Comment:
Can we move this to SqliteQueryInterfaceTypeScript?

Reply:
I've updated the abstract function to support array of sql query strings which removes the need for this override.